content/docs/0.6.0/reference/microk8s/index.md

```diff
@@ -50,8 +50,7 @@ If you are using Ubuntu, you can install Keptn using the following commands:
   sudo microk8s.enable dns
   sudo microk8s.enable dns ingress
   sudo microk8s.enable storage
-  sudo /snap/bin/microk8s.config > kubeconfig
-  export KUBECONFIG=./kubeconfig
+  sudo /snap/bin/microk8s.config > /home/$USER/.kube/config
```

Comment:
This overwrites your existing Kubernetes configuration, and you will lose access to all previously connected clusters.